File: server/src/main/java/me/iwareq/college/controller/ExampleController.java
package me.iwareq.college.controller;

import org.springframework.web.bind.annotation.CrossOrigin;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@CrossOrigin
@RestController
@RequestMapping("/api/v1/")
public class ExampleController {

	@GetMapping("/example")
	public String getExample() {
		return "Да, правильно, оно работает";
	}
}

File: server/src/main/kotlin/me/iwareq/college/controller/ExampleController.kt
package me.iwareq.college.controller

import org.springframework.web.bind.annotation.CrossOrigin
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@CrossOrigin("http://localhost:3000")
@RestController
@RequestMapping("/api/v1/example")
class ExampleController {

	@GetMapping
	fun getExample() = "Да, правильно, оно работает"
}
